apitools/org.eclipse.pde.api.tools/src/org/eclipse/pde/api/tools/internal/search/XmlReferenceDescriptorWriter.java

```diff
@@ -84,10 +83,11 @@ public class XmlReferenceDescriptorWriter {
 	 *            to write the reports to
 	 * @param debug if debugging infos should be written out to the console
 	 */
+	@SuppressWarnings("restriction")
```

Comment:
using a local variable could move the suppress to the actual line of code.

Reply:
Unfortunately the parser field is also used below and since this method is relatively small in scope I skipped the extra variable in this case for simpler code.